File: AtlasWeb/Core/AtlasWebSocketManager/Containers/AtlasNotificationContainer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Core.AtlasWebSocketManager.Containers
{
    public class AtlasNotificationContainer : AtlasWebSocketContainer, I_AtlasWebSocketContainer
    {
        public AtlasNotificationType notificationType { get; set; }
        public bool isRead { get; set; }

        public AtlasNotificationContainer() : base()
        {

        }
    }
}

File: AtlasWeb/Core/wwwroot/app/Fw/Models/AtlasNotificationcontainer.ts
import { AtlasWebSocketContainer } from "./AtlasWebSocketContainer";

export enum AtlasNotificationType {
    Unknown = 0,
    Warning = 1,
    Info = 2,
    Error = 3,
    Message = 4
}

export class AtlasNotificationContainer extends AtlasWebSocketContainer {
    public notificationType: AtlasNotificationType;
    public isRead: boolean;
}